js/tiles.js

```diff
@@ -974,11 +983,39 @@ const door = {
         }
     }),
     threeHeads: Object.assign({}, OpenDoors, {
+    	canEnterFromTheRight() {
```

Comment:
You can change all these functions but please also change the look. Currently, it can be entered from left, top, bottom.

Reply:
@saarthakchats Please respond to this. I just want your view on this.